generator.sh

```diff
@@ -73,39 +74,83 @@ done
 
 ios_ci_support=0
 android_ci_support=0
-tab_layout="false"
 buddybuild_support=0
+enable_preview="false"
+tab_layout="false"
+project_type="adaptive.js"
 
-read -p'--> On iOS, do you want continuous integration? (y/n) ' -n 1 -r
+read -p'➞ On iOS, do you want continuous integration? (y/n) ' -n 1 -r
 echo
 if [[ $REPLY =~ ^[Yy]$ ]] ; then
-    echo '    ↳ To setup iOS continuous integration, see README.md.'
+    echo '  ↳ To setup iOS continuous integration, see README.md.'
     ios_ci_support=1
 fi
 
-read -p "--> On Android, do you want continuous integration? (y/n) " -n 1 -r
+read -p "➞ On Android, do you want continuous integration? (y/n) " -n 1 -r
 echo
 if [[ $REPLY =~ ^[Yy]$ ]] ; then
-    echo '    ↳ To setup Android continuous integration, see README.md.'
+    echo '  ↳ To setup Android continuous integration, see README.md.'
     android_ci_support=1
 fi
 
 if [[ $ios_ci_support -ne 1 && $android_ci_support -ne 1 ]]; then
-    echo '    ↳ Skipping buddybuild integration because continuous integration was not included'
+    echo '  ↳ Skipping buddybuild integration because continuous integration was not included'
 else
-    read -p "--> Do you want buddybuild support? (y/n) " -n 1 -r
+    read -p "➞ Do you want buddybuild support? (y/n) " -n 1 -r
     echo
     if [[ $REPLY =~ ^[Yy]$ ]] ; then
         buddybuild_support=1
     fi
 fi
 
-read -p'--> Do you want to use a tab layout (otherwise a drawer layout will be setup)? (y/n) ' -n 1 -r
+read -p '➞ Do you want to enable Mobify preview? (y/n) ' -n 1 -r
+echo
+if [[ ! $REPLY =~ ^[Yy]$ ]] ; then
+    echo '  ↳ To enable Mobify preview later, see README.md.'
+else
+    REPLY=
+    enable_preview="true"
+    
+    echo "➞ What type of project is this?"
+    echo "    1. Mobify.js (legacy)"
+    echo "    2. Adaptive.js (legacy)"
+    echo "    3. Progressive web"
```

Comment:
Should this be in here? As Astro does not currently work with progressive. The way this question is setup it will be really easy to add in the future.